test/writer.fb2

```diff
@@ -261,23 +261,23 @@
 <p>Nested</p>
 </title>
 <p>• Tab</p>
-<p>◦ Tab</p>
-<p>* Tab</p>
+<p>• • Tab</p>
+<p>• • • Tab</p>
 <p>Here’s another:</p>
 <p>1. First</p>
 <p>2. Second:</p>
-<p>   • Fee</p>
-<p>   • Fie</p>
-<p>   • Foe</p>
+<p>2. • Fee</p>
```

Comment:
I don't know what this looks like in the reader, but it seems like odd output (not natural looking).
Is there a way to force indentation -- perhaps by using nonbreaking spaces?

Reply:
Indentation with nonbreaking spaces, as used for bullet lists, does not work in FBReader and fbless. I don't know why they are used in the first place, maybe there are some readers which display them. Besides that, most readers don't use monospace fonts, so indentation will not be the same even if spaces are displayed. So I decided to just follow what pandoc does for ordered lists, at least it works even if ugly.